starter-core/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,21 @@
+dependencies:
```

Comment:
If understand correctly @alvarosanchez what you are suggesting is to replace the yaml configuration with a pom.xml file such as:

```xml
<project>
  <dependencies>
    <dependency>
      <groupId>org.mybatis</groupId>
      <artifactId>mybatis</artifactId>
      <version>3.4.6</version>
    </dependency>
    <dependency>
      <groupId>org.gebish</groupId>
      <artifactId>geb-junit5</artifactId>
      <version>4.0</version>
    </dependency>
    <dependency>
      <groupId>org.seleniumhq.selenium</groupId>
      <artifactId>selenium-firefox-driver</artifactId>
      <version>3.141.59</version>
    </dependency>
    <dependency>
      <groupId>org.seleniumhq.selenium</groupId>
      <artifactId>selenium-support</artifactId>
      <version>3.141.59</version>
    </dependency>
  </dependencies>
</project>
```
is that correct?

The benefit will be that dependabot will create prs automatically to such a file, right?

We will need to support multiple pom files for feature depency version. As the guides project will need to have its own pom with dependencies such as mybatis, sendgrid, spreadsheetbuilder etc.

Reply:
Correct, that is what I suggest.

Then we could point Dependabot to that folder: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#directory

Reply:
does `dependabot` needs the file to be named `pom.xml`. ? 

should we place them in the `META-INF`? 

For example in starter-core: 

`starter-core/src/main/resources/META-INF/starter-dependencies/core/pom.xml`

and in guides: 

`buildSrc/src/main/resources/META-INF/starter-dependencies/guides/pom.xml`

I could have a factory which reads every pom.xml in the `META-INF/starter-dependency` and creates `MavenCoordinate` beans.

Reply:
Yes, I think it needs to be named `pom.xml`. The directory can be whatever you want, though

Reply:
I've used a pom.xml